step-by-step-general/04-check-compat.md

```diff
@@ -2,22 +2,18 @@
 
 We are going to check if this device is supported.
 
-1. launch ReVanced Manager.
+1. Launch ReVanced Manager. Ignore any updates it claimed it knew.
 ==- Image for reference
 ![launch rvm](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/040-first_launch_manager.jpg?raw=true)
 ===
 2. Go to the settings tab and scroll down until you find "About"
 ==- Image for reference
 ![check about](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/050-check_about.jpg?raw=true)
 ===
-3. Check that `Arch` had at least 1 entry that looks like `arm64-v8a`
-4. If you only got something that said `armv7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-5. If you got `armv8`, you are probably fine
-6. Check that your Android version is at least 8
-7. If you only got something that said `armv7` or lower, you may not be able to patch on your phone. You can patch it on another device or a computer with [ReVanced CLI](/06-revanced-cli.md).
-8. If you made it here, you can probably patch on your phone with ReVanced Manager
+3. Check that `Arch` says `arm64-v8a`
+4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armv7` not supported?
+==- What does `arch` mean? Why is `armeabi-v7a` not supported?
```

Comment:
`armeabi-v7a` means emulated armv7 (32 bit). This appears on devices that has `armv8` but wanted to retain compatibility with older apks. 

`armv7` is the intended.

Reply:
I believe the Manager says "armeabi-v7a" on armv7 devices. The guide should refer to it by whatever the manager calls it, since users don't know that armv8 = arm64-v8a  or that armv7 = armeabi-v7a.

This screenshot is old, but I'm pretty sure it still says this and not "armv7"
![SmartSelect_20240708_105353_1.png](https://github.com/SodaWithoutSparkles/revanced-troubleshooting-guide/assets/84587632/4cc0ac09-5e89-40c1-96e4-ce906ac439ab)

If the manager did say "armv7" it would make sense for the guide to refer to it as such.